src/preamble_minimal.js

```diff
@@ -0,0 +1,135 @@
+#if SEPARATE_ASM && ASSERTIONS && WASM == 0 && MODULARIZE
+if (!({{{ASM_MODULE_NAME}}})) throw 'Must load asm.js Module in to variable {{{ASM_MODULE_NAME}}} before adding compiled output .js script to the DOM';
+#endif
+
+#include "runtime_safe_heap.js"
+
+#if ASSERTIONS
```

Comment:
these functions are shared with the other code, how about including a single js file for both?

Reply:
I did not want to share these because these reflect things that I want to be removed in the future from preamble_minimal, rather than something to build up abstractions towards. Sharing the code would move it in the direction where it would be harder to remove, copying all of these here makes it more visible what remains.

The other files with runtime_strings.js/runtime_sab_polyfill.js etc. are more stable items that are safer to share, because I don't expect there to be big changes to them.